### PR TITLE
Update component import in README.md

### PR DESCRIPTION
**Pull request by paschaldev**
`components` are stored in generated `api` folder not `server`

----

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@
 Sometimes your app needs to fetch information from a third-party API that is slow or costs money. Caching can help! This is a Convex component that can cache the results of expensive functions and set an optional TTL. Expired entries are cleaned up via a cron job once a day. The cache key is the `ActionCache`'s name (defaults to function name) and the arguments to the action that generates the cache values.
 
 ```ts
-import { Client } from "@convex-dev/cache";
```
Comment (paschaldev) [inline README.md:12]: `Client` import is not referenced anywhere
`@convex-dev/cache` package does not exist

File: README.md [hunk 2]
```diff
@@ -9,8 +9,8 @@
...
-import { action, components } from "./_generated/server";
```
Comment (paschaldev) [inline README.md:13]: `components` live in `./_generated/api`

File: README.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ Finally, create a new `ActionCache` with optional name and expiration within you
   - If an `ttl` is provided, expired cache entries are deleted when they are retrieved and in a daily cron job.
 
 ```ts
-import { ActionCache } from "@convex-dev/cache";
```
Comment (paschaldev) [inline README.md:75]: `ActionCache` is imported from `@convex-dev/action-cache`
`@convex-dev/cache` package does not exist

**Review by emmaling27 — APPROVED** [review]
thank you for fixing! these snippets were out of date

**Pull request merged**